src/main/java/com/internship/backend/BackendApplication.java

```diff
@@ -7,6 +7,7 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.annotation.Bean;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
```

Comment:
Could you check the imports, and remove the unused ones? Not sure if we need EnableWebSecurity import here.